### DISCUSSION TITLE
DiME installation

### DISCUSSION THREAD

**Discussion by JunejaAmmy** [CURENT/andes#282] [Q&A] [answered]
Hi,

I was following [277](https://github.com/cuihantao/andes/discussions/277) and also tried to install the DiME. I went through the README of [TheHashTableSlasher](https://github.com/TheHashTableSlasher/dime/blob/master/README.md). But faced below error.

Please help!

![image](https://user-images.githubusercontent.com/97614806/156032914-d7875d4f-e12d-45f7-8fc2-d51347b02b64.png)

**Comment by cuihantao** [comment]
It only says the installation was not successful. You will need to double-check for error messages while installing DiME. It might be better to post the issues to the DiME repository.

**Comment by JunejaAmmy** [comment, marked as answer]
I was able to install DiME by downloading its code.
